### PR TITLE
added images to IngredientsSearch and Search

### PR DESCRIPTION
**Pull request by Darkflair**

### CODE REVIEW

File: app/src/main/java/com/example/pantrypal/apiTools/MealDBRecipe.java [hunk 1]
```diff
@@ -25,7 +25,7 @@ public MealDBRecipe(int id,
                         List<String> tags,
                         String youtubeLink,
                         List<String> ingredients) {
-        super(name, String.valueOf(id), ingredients);
```
Comment (TomaszTB) [inline app/src/main/java/com/example/pantrypal/apiTools/MealDBRecipe.java:28]: Now that you're passing in the ImageURL to the super class, there's no need to store it in this class as well. Could you remove the ImageURL field from this class and replace the calls to `getImageURL` with `ResultsRecipe`'s `getImageUrl`?

File: app/src/main/java/com/example/pantrypal/activities/Search.java [hunk 2]
```diff
@@ -3,11 +3,13 @@
 import static android.widget.Toast.makeText;
 
 import android.content.Intent;
+import android.net.Uri;
```
Comment (TomaszTB) [inline app/src/main/java/com/example/pantrypal/activities/Search.java]: There are some unused import here. Could you do Ctrl + Alt + O to clean them up?

File: app/src/main/java/com/example/pantrypal/ResultsRecipe.java [hunk 3]
```diff
@@ -1,5 +1,7 @@
 package com.example.pantrypal;
 
+import android.net.Uri;
```
Comment (TomaszTB) [inline app/src/main/java/com/example/pantrypal/ResultsRecipe.java]: This import isn't used. Could you remove it?

**Review by TomaszTB — COMMENTED** [review]
Functions great! Just gotta clean the code up a bit

**Comment by Darkflair** [comment]
I might need your help with changed the calls for getImageUrl. It kinda breaks everything when I tried to fix it

**Comment by TomaszTB** [comment]
I made the change. Everything seems to work fine for me. Could you double check it?

**Pull request merged**